loom/script/reflection/lsType.cpp

```diff
@@ -38,7 +38,6 @@ Assembly *Type::getAssembly()
 
 void Type::addMember(MemberInfo *member)
 {
-    LOOM_ALLOCATOR_VERIFY(member);
```

Comment:
VERIFY just checks that the pointer is valid when checking is enabled, this was most likely left over from some just-to-be-sure debug code and is as such no longer needed as it didn't seem to show any problems.